rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -3,26 +3,33 @@ class QuotingPanel {
     this.polygons = {};
     this.totalAreaInSqFt = null;
     this.totalDue = null;
+
+    this.handleQuoteSubmit = this.handleQuoteSubmit.bind(this);
+    this.handlePolygonChanged = this.handlePolygonChanged.bind(this);
+    this.addListeners();
   }
 
   showAddress(geocodedAdress) {
     if (geocodedAdress) {
       let addressNode = document.getElementById("displayAddress");
-      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
       let splitAddress = geocodedAdress.split(",");
+      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
       splitAddress.forEach(element => {
         let div = document.createElement("div");
         div.innerText = element;
         addressNode.appendChild(div);
       });
       addressNode.classList.remove("hidden");
+
+      let addressNodeInModal = document.getElementById("addressModal");
```

Comment:
If you are not planning on changing this value you can use `const` over `let`. The default type should always be `const`, and then switch to `let` only when necessary.